syntaxes/odoc.json

```
@@ -0,0 +1,171 @@
+          "contentName": "source.embedded.odoc",
+          "patterns": [{ "include": "source.ocaml" }]
+        },
+        {
+          "comment": "label",
+          "begin": "\\{([[:digit:]]+)\\:[[:space:]]?[a-z]+",
+          "end": "\\}",
+          "beginCaptures": {
+            "1": { "name": "variable.other.declaration.odoc" }
+          },
+          "patterns": [{ "include": "#markup" }]
+        },
+        {
+          "comment": "section header",
+          "begin": "\\{([[:digit:]]+)",
+          "end": "\\}",
+          "beginCaptures": {
+            "1": { "name": "constant.language.odoc" }
+          },
+          "patterns": [{ "include": "#markup" }]
+        },
+        {
+          "comment": "text in bold",
+          "begin": "\\{b\\b",
+          "end": "\\}",
+          "contentName": "strong",
+          "patterns": [{ "include": "#markup" }]
+        },
+        {
+          "comment": "emphasize text",
+          "begin": "\\{e\\b",
+          "end": "\\}",
+          "contentName": "emphasis",
+          "patterns": [{ "include": "#markup" }]
+        },
+        {
+          "comment": "text in italic",
+          "begin": "\\{i\\b",
+          "end": "\\}",
+          "contentName": "emphasis",
+          "patterns": [{ "include": "#markup" }]
+        },
+        {
+          "comment": "center text",
+          "begin": "\\{C\\b",
+          "end": "\\}",
+          "patterns": [{ "include": "#markup" }]
+        },
+        {
+          "comment": "left align text",
+          "begin": "\\{L\\b",
+          "end": "\\}",
+          "patterns": [{ "include": "#markup" }]
+        },
+        {
+          "comment": "right align text",
+          "begin": "\\{R\\b",
+          "end": "\\}",
+          "patterns": [{ "include": "#markup" }]
+        },
+        {
+          "comment": "unordered list",
+          "begin": "\\{ul\\b",
+          "end": "\\}",
+          "patterns": [{ "include": "#markup" }]
+        },
+        {
+          "comment": "ordered list",
+          "begin": "\\{ol\\b",
+          "end": "\\}",
+          "patterns": [{ "include": "#markup" }]
+        },
+        {
+          "comment": "link address text",
+          "begin": "\\{(?=\\{\\:)",
+          "end": "\\}",
+          "contentName": "string.other.odoc",
+          "patterns": [{
+            "begin": "\\{\\:",
+            "end": "\\}"
+        },
+        { "include":" #markup" }]
+        },
+        {
+        "comment": "string in source code",
+        "begin": "\\[",
+        "end": "\\]",
+        "contentName": "string.other.odoc",
+        "patterns": [{ "include": "#markup" }]
+      },
+      {
+        "comment": "string in preformatted source code",
+        "begin": "\\{\\[",
+        "end": "\\]\\}",
+        "contentName": "string.other.odoc",
+        "patterns": [{ "include": "#markup" }]
+      },
+      {
+        "comment": "string in verbatim",
+        "begin": "\\{v\\b",
+        "end": "v\\}",
```

Since this regex pattern beings with a letter it needs a boundary too:
```suggestion
        "end": "\\bv\\}",
```